### DISCUSSION TITLE
Kubectl edit issue

### DISCUSSION THREAD

**Discussion by PCesteban** [LunarVim/LunarVim#3692] [Q&A]
I have set lvim as my default editor. When I run `kubectl edit` I can edit the file in lvim, but when I save and exit I receive the following error:

```
$ kubectl edit deployment -n <pod-namespace>
error: there was a problem with the editor "lvim"
```

Does anyone have experienced this error before?

**Comment by LostInTheLogs** [comment]
set editor to full path of lvim

**Reply by PCesteban** [reply]
Hi, sorry for the late response,
Same issue: 
`error: there was a problem with the editor "/home/et/.local/bin/lvim"`

**Reply by LostInTheLogs** [reply]
I've got no idea what's the problem in that case, does it work with neovim?

**Reply by PCesteban** [reply]
Yes, it works without problem

**Reply by LostInTheLogs** [reply]
what if you put a bash script in `/home/et/.local/bin` that just runs `exec nvim "$@"`, and try that script as the editor

**Reply by PCesteban** [reply]
It doesn't even open the editor, it throws the error right away.
`error: there was a problem with the editor "/home/et/.local/bin/test.sh"`

In the case of lvim, it opens the editor and I can edit the file inside lvim normally, but when I save and quit the file it doesn't save my changes and that error appears.

**Reply by PCesteban** [reply]
Hi @LostNeophyte! Any update on this?

**Reply by LostInTheLogs** [reply]
no, I know nothing about kubectl, idk why a bash script wouldn't work

**Reply by PCesteban** [reply]
I see, I will open an issue, maybe this could be an improvement to lvim.
